src/main/java/com/diffmin/util/GumtreeSpoonUtil.java

```diff
@@ -14,11 +13,10 @@
 import spoon.reflect.visitor.DefaultJavaPrettyPrinter;
 import spoon.reflect.visitor.PrettyPrinter;
 
-/** Computes the edit script and uses it to patch the previous version. */
-public class App {
-
+/** Utility class for interacting with the {@link CtModel} and computing {@link Diff}. */
+public class GumtreeSpoonUtil {
     /** Override constructor to prevent instantiating of this class (RSPEC-1118). */
-    private App() {
+    private GumtreeSpoonUtil() {
```

Comment:
This class is difficult to name because it conceptually contains two things:

1. Utility methods for Spoon
2. A single utility method for gumtree-spoon

Since the vast majority of the class is concerned only with Spoon stuff, I would name it `SpoonUtil`. The `computeDiff` method should arguably be moved somewhere else, but that's a concern for some other PR.

Reply:
> Utility methods for Spoon
> A single utility method for gumtree-spoon

Exactly my thoughts too. I initially named it `SpoonModelUtil` but realised the second point and renamed it to `GumtreeSpoonUtil` because GumTreeSpoon contains "Spoon" as well. 😅